Products/GenericSetup/registry.py

```diff
@@ -644,6 +644,10 @@ def __init__(self):
     def getProfileInfo(self, profile_id, for_=None):
         """ See IProfileRegistry.
         """
+        if profile_id.startswith("profile-"):
+            profile_id = profile_id[len('profile-'):]
```

Comment:
Well, looking more closely, your comment was about two lines that both had `profile-`, but after those lines are two similar lines with `snapshot-`. I guess that was your point, but it got lost because I did not see those last two lines on github when reading your comment.
I'll make the code a bit clearer. :-)